src/main/java/seedu/trippie/Parser.java

```diff
@@ -1,14 +1,6 @@
 package seedu.trippie;
 
-import seedu.trippie.command.AddExpenseCommand;
-import seedu.trippie.command.AddPlaceCommand;
-import seedu.trippie.command.Command;
-import seedu.trippie.command.DeleteExpenditureCommand;
-import seedu.trippie.command.DeletePlaceCommand;
-import seedu.trippie.command.DisplayTotalExpenditureCommand;
-import seedu.trippie.command.ExitCommand;
-import seedu.trippie.command.HelpCommand;
-import seedu.trippie.command.ListExpenditureCommand;
+import seedu.trippie.command.*;
```

Comment:
It would be better to mention specifically what are being imported instead of using wildcard `*`